airbyte-webapp/src/components/connection/ConnectionForm/Section.module.scss

```diff
@@ -1,5 +1,11 @@
 @use "scss/variables";
 
+.flexHeight {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+}
+
```

Comment:
Do we really want to stretch the stream table to the bottom if it only has a few rows? Looks kind of strange IMO:
![image](https://user-images.githubusercontent.com/7550957/214830832-e211091f-bc70-43e1-8764-6769180f1c79.png)

Reply:
That's how we planned it out in the design.

Reply:
Looks a bit empty to me, but ok 🙂